src/drivers/nuopc/cime_config/buildnml

```diff
@@ -401,10 +401,14 @@ def buildnml(case, caseroot, component):
 
     # copy drv_in, drv_flds_in, cesm.runconfig and all *modio* fiels to rundir
     rundir = case.get_value("RUNDIR")
-
     shutil.copy(os.path.join(confdir,"drv_in"), rundir)
     shutil.copy(os.path.join(confdir,"cesm.runconfig"), rundir)
 
+    # copy fd.yaml to rundir
+    config_dir = os.path.dirname(definition_file[0])
```

Comment:
This looks like you are using config_files.xml but then ignoring it and hard coding a path and file name here.